apps/els_lsp/src/els_index_buffer.erl

```diff
@@ -0,0 +1,110 @@
+%% @doc Buffer textedits to avoid spamming indexing for every keystroke
+%%
+%% FIXME: Currently implemented as a simple process.
+%%        Might be nicer to rewrite this as a gen_server
```

Comment:
I would definitely rewrite this as a `gen_server`, but we can do that as a follow up.